stores/sql/sqlite/main.go

```diff
@@ -89,10 +127,16 @@ func (b *MainDatabase) MakeDirsForPath(tx sql.Tx, path string) (uint, error) {
 	return dirID, nil
 }
 
-func (b *MainDatabase) Migrate() error {
-	return sql.PerformMigrations(b, migrationsFs, "main", sql.MainMigrations(b, migrationsFs, b.log))
-}
-
-func (b *MainDatabase) Version(_ context.Context) (string, string, error) {
-	return version(b.db)
+func (tx *MainDatabaseTx) RenameObject(bucket, keyOld, keyNew string, dirID uint) error {
+	resp, err := tx.Exec(`UPDATE objects SET object_id = ?, db_directory_id = ? WHERE object_id = ? AND db_bucket_id = (SELECT id FROM buckets WHERE buckets.name = ?)`, keyNew, dirID, keyOld, bucket)
+	if err != nil && strings.Contains(err.Error(), "UNIQUE constraint failed") {
```

Comment:
could've queried it with `SELECT 1 FROM ...` instead of checking for the constraint string